setup.py

```diff
@@ -107,17 +108,35 @@ def get_extensions():
     extensions_cuda_dir = os.path.join(extensions_dir, "cuda")
     cuda_sources = list(glob.glob(os.path.join(extensions_cuda_dir, "**/*.cu"), recursive=True))
 
-    if use_cuda:
+    extensions_hip_dir = os.path.join(extensions_dir, "cuda", "fp6_llm")
+    hip_sources = list(glob.glob(os.path.join(extensions_hip_dir, "*.cu"), recursive=True))
```

Comment:
seems file for now, but we might have other extensions other than ".cu" later?

Reply:
yes, once we enable all the extensions - we can remove hip_sources and start using cuda_sources directly.